pallets/usage/src/tests.rs

```diff
@@ -112,6 +112,27 @@ fn wait_list_contents() -> Vec<(StoredDispatch, u32)> {
     .collect()
 }
 
+fn decode_validate_usage_call(encoded: &[u8]) -> crate::Call<Test> {
+    let mut encoded = <&[u8]>::clone(&encoded);
+    let extrinsic: Extrinsic = Decode::decode(&mut encoded).unwrap();
+
+    let call = extrinsic.call;
+    let inner = match call {
+        mock::Call::Usage(inner) => inner,
+        _ => unreachable!(),
+    };
+
+    assert!(
+        <Usage as sp_runtime::traits::ValidateUnsigned>::validate_unsigned(
```

Comment:
Isn't it better to import the trait?
```suggestion
        <Usage as ValidateUnsigned>::validate_unsigned(
```

Reply:
yep, ok